Vagrantfile

```diff
@@ -3,7 +3,7 @@
 
 Vagrant.require_version '>= 1.4.1'
 
-%w[vagrant-ohai2 vagrant-omnibus].each do |plugin|
```

Comment:
```
==> c2_boot1: Running chef-client...
==> c2_boot1: [2018-10-24T21:40:58+00:00] FATAL: Configuration error NoMethodError: undefined method `<<' for nil:NilClass
==> c2_boot1: [2018-10-24T21:40:58+00:00] FATAL:   /tmp/vagrant-chef/client.rb:32:in `from_string'
==> c2_boot1: [2018-10-24T21:40:58+00:00] FATAL: Aborting due to error in '/tmp/vagrant-chef/client.rb'
```

@som3guy  I keep running into this on your branch and I believe you had the fix for this in vagrant-ohai2. Right?
https://github.com/avishai-ish-shalom/vagrant-ohai/pull/21/files

Reply:
There's no way to enable or disable vagrant plugins.  If you have it installed, it will get used.  I had to uninstall the vagrant-ohai plugin to get the vagrant nodes to come up on this branch. With the latest changes, the vagrant-ohai2 plugin is also unnecessary.

Reply:
Yep.. uninstalling fixed it.